timeslots/handler.go

```diff
@@ -11,14 +11,27 @@ type TimeslotData struct {
 	Time    []TimePeriod `json:"time"`
 }
 
+// time payload format that works
+// "time": [{"From": "2014-11-12T11:45:26.371Z", "To": "2017-11-12T11:45:26.371Z"}]
 func CreateTimeslots(w http.ResponseWriter, r *http.Request) {
 	var timeslotsData TimeslotData
 	json.NewDecoder(r.Body).Decode(&timeslotsData)
 
 	repo := NewRepository(db.DbInstance)
 
+	for _, timePeriod := range timeslotsData.Time {
+		if !isTimePeriodValid(timePeriod) {
+			return
+		}
+	}
+
+	sortByTimeFrom(timeslotsData.Time)
```

Comment:
run overlapping validation here. and sorting might be a part of the validation logic.